docs/generated/errors-list.md

```diff
@@ -88,7 +88,7 @@ The `galasactl` tool can generate the following errors:
 - GAL1085E: Programming logic error. ReadTextFile operation on embedded file system failed. Reason is {}
 - GAL1086E: Communications problem between the command-line tool and the target Galasa Ecosystem. The tool could not retrieve the list of valid result names. Reason: '{}'
 - GAL1087E: Invalid '--result' parameter value: {}. The possible result values currently in the Ecosystem Result Archive Store (RAS) are: {}
-- GAL1088E: --active and --result must not be used at the same time, they are mutually exclusive.
```

Comment:
This looks a little odd. I'm not sure why this is deleted and replaced.

Reply:
deferring this to a separate PR